client/stylesheets/abstracts/_colors.scss

```diff
@@ -63,5 +63,6 @@ $muriel-gray-50: #e1e2e2;
 $muriel-gray-900: #1a1a1a;
 $muriel-gray-300: #969ca1;
 $muriel-gray-500: #636d75;
+$muriel-gray-600: #50575d;
```

Comment:
I've merged the color PR, so you can drop this addition.

Reply:
Dropped!